uninstall.sh

```diff
@@ -5,27 +5,16 @@
 # This script tries to remove as much as reasonable to allow for re-testing
 # bootstrap.sh multiple times on the same host
 
-remove_other() {
-    # Remove Sentry and dependencies
-    rm -rf ~/.sentry
-    rm -rf ~/code/sentry/{.venv,node_modules}
-    rm -rf ~/code/getsentry/{.venv,node_modules}
-    [ -f ~/.zprofile ] && rm ~/.zprofile
-}
+if [ -n "$CI" ]; then
+  # macos in GHA is intel-only, so make sure we remove its brew
+  sudo rm -rf /usr/local/Homebrew
+  sudo rm -f /usr/local/bin/brew
+fi
 
-remove_brew_setup() {
-    # Removes all packages
-    brew list -1 | xargs brew rm
-    # Execute the official uninstall command
-    /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/uninstall.sh)"
-    # This is not removed by the script.
-    [ -d /opt/homebrew ] && sudo rm -rf /opt/homebrew
-    [ -d /usr/local/bin ] && sudo rm -rf /usr/local/bin
-}
+sudo rm -rf /opt/homebrew
 
-# Uninstall brew
-command -v brew &>/dev/null && remove_brew_setup
-
-remove_other
-
-echo "Successfully uninstalled brew and other"
+rm -rf ~/.pyenv
+rm -rf ~/.sentry
+rm -rf ~/code/sentry/{.venv,node_modules}
+rm -rf ~/code/getsentry/{.venv,node_modules}
+rm -f ~/.profile ~/.zprofile ~/.zshrc
```

Comment:
hopefully nobody runs this and is surprised about their zshrc lol

Reply:
yeah that is not good

Reply:
i'll add something to warn, and also move it to backup instead of deleting, but i don't _expect_ anyone to run this